### DISCUSSION TITLE
Faster-Whisper is running on CPU - Nvidia P4000

### DISCUSSION THREAD

**Discussion by Giak1234** [Purfview/whisper-standalone-win#192] [Q&A]
Good morning,
may I ask for suggestions on which command to run to get a better performing transcription time?
I currently run the following command to translate a folder containing various audio: whisper-faster.exe E:\audio --language=en --model=medium --output_dir=C:\transcripts --output_format=txt
The job screen indicates runing on : cpu ... but to work with the GPU (I have an nVidia with 8GB of memory)? What setup do you recommend? Thank you very much.

Translated with DeepL.com (free version)

**Comment by Purfview** [comment]
>may I ask for suggestions on which command to run to get a better performing transcription time?

Try these options:
`--beam_size 1`
`--compute_type float32`

For English you can try Distil models ->https://github.com/Purfview/whisper-standalone-win/discussions/180

>The job screen indicates runing on : cpu ... but to work with the GPU (I have an nVidia with 8GB of memory)?

What is your exact GPU model? And what NVIDIA driver version?

**Reply by Giak1234** [reply]
Thanks @Purfview ...
I will try your suggestion and update you later :-)
nVidia P4000 with the latest driver 551.52 

![nVidia](https://github.com/Purfview/whisper-standalone-win/assets/17331993/6a0502a1-7682-4548-8b0b-a2e3d88b92b7)

**Reply by Purfview** [reply]
I think it should work on this GPU, I dunno why it's not detected as CUDA capable.

Try `--device cuda`.

**Reply by ghost** [reply]
@Giak1234 enable Hardware-accelerated GPU scheduling from graphics settings on windows. Disable sysmemfallback on nvidia control manager panel and disable cuda force p2 state with nvidia profile inspector

**Reply by Giak1234** [reply]
Good afternoon everyone!

I detected this one problem ... **cudnn_ops_infer64_8.dll**

![nVidia1](https://github.com/Purfview/whisper-standalone-win/assets/17331993/097f8359-c188-4921-86a9-5a04e8eff6d8)

To solve it I have already installed the cuDNN library (https://developer.nvidia.com/cudnn) and tried to install the CUDA Toolkit without success (12.3, 12.1.1, 11.0.2 and 10.1.243 )!

![nVidia2](https://github.com/Purfview/whisper-standalone-win/assets/17331993/956ec495-5023-42f3-b113-696652d5933c)

I don't understand where the problem is :-( ... **suggestions**?

**Reply by ghost** [reply]
@Giak1234 https://github.com/Purfview/whisper-standalone-win/releases/tag/libs Download libs from there and put them on the same folder as whisper-faster.exe

**Reply by Purfview** [reply]
> I don't understand where the problem is :-( ... suggestions?

CUDA 10.x is not supported, you need cuBLAS and cuDNN 8.x libs for CUDA 11.x

**Reply by Giak1234** [reply]
Thanks @toprak ... now works ;-)
I still have to install the CUDA Toolkit ... I cannot fail ... i will update you.
